test/e2e/app-dir/app/flying-shuttle.test.ts

```diff
@@ -35,6 +35,12 @@ describe('should output updated trace files', () => {
     )
 
     expect(deploymentsTrace.fileHashes).toBeTruthy()
+
+    const deploymentsFileHashKeys = Object.keys(deploymentsTrace.fileHashes)
+    expect(
+      deploymentsFileHashKeys.filter((item) => item.includes('/layout')).length
+    ).toBe(3)
```

Comment:
just to make sure I'm understanding this correctly: this is 3 because of the root layout, the layout in `/dashboard` and the layout in `/dashboard/deployments`?

Reply:
Yea exactly